### PR TITLE
561 remove closures

### PR DESCRIPTION
**Pull request by SimplyAhmazing**
This PR has several impacts,

* Removes robot singleton
* Removes command queue closures, api calls are direct calls to driver
* Removes upload to Jupyter feature (since commands are no longer stored and cannot be serialized)
* Creates a `utils/state.py` and refactors methods for querying robot state from flask app
* Removes JSON importer
* Removes threaded test for pausing a running protocol as commands are no longer in a queue. This needs to be re-thought
* Removes `enqueue=True` from documentation

### CODE REVIEW

File: api/opentrons/server/main.py [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
 
-import dill
 import flask
 from flask import Flask, render_template, request
 from flask_socketio import SocketIO
 from flask_cors import CORS
 
-from opentrons import robot, Robot, containers, instruments
-from opentrons.util import trace, environment
+from opentrons import robot, Robot, instruments, containers  # NOQA
```
Comment (SimplyAhmazing) [inline api/opentrons/server/main.py]: I just remembered this is for having these imports in `globals()` so when we `eval` a protocol we provide the imports to the users protocol.

File: api/opentrons/server/main.py [hunk 2]
```diff
@@ -1098,7 +832,7 @@ def start():
 
     socketio.run(
         app,
-        debug=False,
+        debug=True,
```
Comment (astaff) [inline api/opentrons/server/main.py]: Change back?

File: api/opentrons/server/tests/test_connect_and_diagnostics.py [hunk 3]
```diff
@@ -59,9 +55,10 @@ def test_connect(self):
         self.assertTrue(response['is_connected'])
         self.assertEquals(response['port'], 'Virtual Smoothie')
 
+    # FIXME
```
Comment (astaff) [inline api/opentrons/server/tests/test_connect_and_diagnostics.py]: Fix or write a comment

File: api/tests/opentrons/integration/foo.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+from opentrons import instruments, containers
```
Comment (astaff) [inline api/tests/opentrons/integration/foo.py]: Remove from git?

**Comment by astaff** [comment]
LGTM

**Pull request merged**